desktop/cmp/treemap/SplitTreeMap.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2019 Extremely Heavy Industries Inc.
+ */
+import React, {Component} from 'react';
+import PT from 'prop-types';
+import {elemFactory, HoistComponent, LayoutSupport} from '@xh/hoist/core';
+import {hframe, vframe, frame} from '@xh/hoist/cmp/layout';
+import {panel} from '@xh/hoist/desktop/cmp/panel';
+import {compact, uniq} from 'lodash';
+
+import {treeMap} from './TreeMap';
+import {SplitTreeMapModel} from './SplitTreeMapModel';
+
+/**
+ * A component which divides data across two TreeMaps.
+ *
+ * @see SplitTreeMapModel
+ */
+@HoistComponent
+@LayoutSupport
+export class SplitTreeMap extends Component {
+
+    static propTypes = {
+        /** Primary component model instance. */
+        model: PT.oneOfType([PT.instanceOf(SplitTreeMapModel), PT.object]).isRequired
+    };
+
+    static modelClass = SplitTreeMapModel;
+
+    baseClassName = 'xh-split-treemap';
+
+    render() {
+        const {model} = this,
+            {primaryMapModel, secondaryMapModel, orientation} = model,
+            errors = uniq(compact([primaryMapModel.error, secondaryMapModel.error])),
+            container = orientation === 'horizontal' ? hframe : vframe;
+
+        return container({
+            className: this.getClassName(),
+            items: errors.length ? this.renderErrors(errors) : this.renderChildMaps(),
+            ...this.getLayoutProps()
+        });
+    }
+
+    renderChildMaps() {
+        const {model} = this,
+            {primaryMapModel, secondaryMapModel, mapTitleFn} = model,
+            pTotal = primaryMapModel.total,
+            sTotal = secondaryMapModel.total;
+
+        let pFlex = 1, sFlex = 1;
+        if (pTotal && sTotal) {
+            pFlex = pTotal / sTotal;
+        } else if (pTotal && !sTotal) {
+            sFlex = 0;
+        } else if (!pTotal && sTotal) {
+            pFlex = 0;
+        }
+
+        return [
```

Comment:
I think we should add some logic here to either round pFlex or compare it against a last-rendered pFlex value to cut down on unwanted jitter and (relatively expensive) resize events.

Currently we're calculating the ratio of the two with full precision based on the totals. This means that even tiny changes to the P&L on one or both sides will cause a resize, even when it's going to be imperceptible. Log the pFlex values and I think this should be clear.

Rounding pFlex to e.g. two dps would likely cut down on a lot of jitter when you had relatively well-balanced maps (i.e. pFlex was close to one).  Comparing it against the last calculated value and simply using that last value when they are e.g. <2% apart or something like that might be a bit more robust.

Here's what the profile looks like with the main data reaction / re-render followed by the debounced resize. I think we can avoid that last big block of work in many cases.

<img width="2430" alt="Screen Shot 2019-08-13 at 2 44 09 PM" src="https://user-images.githubusercontent.com/225806/62979580-15a1f800-bdd9-11e9-9853-4c911d465a0d.png">